docs/servo_power_monitor/src/dataRecoder.ts

```diff
@@ -0,0 +1,138 @@
+import {powerGraph} from './powerGraph';
+import {operatePort} from './operatePort';
+import {histogram} from './histogram';
+
+export class dataRecoder {
```

Comment:
I think we should rename `DataRecorder` to `DataController` or something. This class should focus on talking with servo and message protocol between servo and this monitor.

The important loops (i.e. `kickWriteLoop()`, `readLoop()`) and related database (i.e. `powerData`) and graph should be moved to `PowerTestController` and this class should provide the simple `read()` and `write()` methods.

Reply:
I think simple `read()` and `write()` methods are in `OperatePort` class, so I renamed `DataRecoder` to `DataParser` and made the class focus on storing servo's output + extract the power data.